fcrepo-configs/src/main/resources/config/clustered-postgesql/jgroups.xml

```diff
@@ -0,0 +1,79 @@
+<!--
+  ~ JBoss, Home of Professional Open Source
+  ~ Copyright 2010 Red Hat Inc. and/or its affiliates and other
+  ~ contributors as indicated by the @author tags. All rights reserved.
+  ~ See the copyright.txt in the distribution for a full listing of
+  ~ individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<!--See: https://docs.jboss.org/author/display/MODE50/ModeShape+in+Java+applications#ModeShapeinJavaapplications-Clustering-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:org:jgroups"
+        xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/jgroups.xsd">
+
+    <TCP bind_port="${jgroups.tcp.port:7800}"
+         bind_addr="${jgroups.tcp.address:0.0.0.0}"
+         recv_buf_size="${tcp.recv_buf_size:5M}"
+         send_buf_size="${tcp.send_buf_size:5M}"
+         max_bundle_size="64K"
+         max_bundle_timeout="30"
+         sock_conn_timeout="3000"
+         timer_type="new3"
+         timer.min_threads="4"
+         timer.max_threads="10"
+         timer.keep_alive_time="3000"
+         timer.queue_max_size="500"
+         thread_pool.enabled="false"
+         oob_thread_pool.enabled="false"
+         port_range="0"/>
+    <JDBC_PING connection_url="jdbc:postgresql://${fcrepo.postgresql.host:localhost}:${fcrepo.postgresql.port:5432}/${fcrepo.postgresql.database:fedora}"
```

Comment:
Hey @orangewolf I'm finally coming back around to this and I noticed two things.
1. You've made the database name configurable, which is fine (and we should probably carry through the other configs).
2. But you have a default database name of `fedora` and we have used `fcrepo` in our [other configs](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-configs/src/main/resources/config/jdbc-postgresql/repository.json#L13). Could you change that please?